src/CommonValidator.js

```diff
@@ -19,6 +19,33 @@ module.exports = {
         let checkSum = line.substr(68, 1);
         return numberRegExp.test(checkSum);
     },
+    checkCheckSum: function(line) {
```

Comment:
Put the contents of this function inside `validCheckSum` as this is now doing a much better job of validating check sum.

Reply:
Done